src/main/resources/META-INF/idea-only.xml

```diff
@@ -1,6 +1,5 @@
 <idea-plugin version="2">
     <extensions defaultExtensionNs="com.intellij">
-        <moduleBuilder builderClass="org.rust.ide.idea.RsModuleBuilder"/>
```

Comment:
Do we really need this? I dropped it because `RsModuleBuilder` has not empty constructor now and IDEA fails to create it with reflection but I don't know the purpose of this extension (kotlin plugin doesn't have it too).
For me, the plugin works well without it:)